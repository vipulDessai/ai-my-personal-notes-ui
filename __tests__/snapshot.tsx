import { render } from "@testing-library/react";

import Home from "../pages/index";
import AddNote from "../pages/add-note";

it("renders homepage unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});

it("renders add note unchanged", () => {
  const { container } = render(<AddNote />);
  expect(container).toMatchSnapshot();
});
