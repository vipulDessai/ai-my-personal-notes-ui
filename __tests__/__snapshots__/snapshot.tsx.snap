// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <header
      class="app-main-header"
    >
      <section />
      <section>
        <nav>
          <button
            aria-label="side menu"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="MenuIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </nav>
      </section>
    </header>
    <main>
      <p>
        Home
      </p>
    </main>
    <footer>
      <a
        class="flex-center"
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <span>
          Powered by
        </span>
        <img
          alt="Vercel"
          class="logo"
          src="/vercel.svg"
        />
      </a>
    </footer>
  </div>
</div>
`;
